packages/fluentui/react-northstar/src/components/Animation/Animation.tsx

```diff
@@ -14,14 +11,15 @@ import * as React from 'react';
 import { ThemeContext } from 'react-fela';
 import { Transition } from 'react-transition-group';
 
-import { childrenExist, StyledComponentProps, commonPropTypes, ChildrenComponentProps } from '../../utils';
+import { childrenExist, commonPropTypes, ChildrenComponentProps } from '../../utils';
 import { ComponentEventHandler, ProviderContextPrepared } from '../../types';
+import useAnimationStyles from './useAnimationStyles';
+
+export { animationClassName } from './useAnimationStyles';
```

Comment:
I would like to avoid reexport there, can we reexport it in `src/index.ts` instead?

Reply:
Moved